products/contao

```diff
@@ -0,0 +1,38 @@
+---
+title: Contao
+category: server-app
+changelogTemplate: "https://github.com/contao/contao/blob/__RELEASE_CYCLE__/CHANGELOG.md"
+auto:
+  - git: https://github.com/contao/contao.git
+
+releases:
+  - releaseCycle: "5.0"
+    releaseDate: 2022-08-18
+    support: 2023-02-14
+    eol: 2023-02-14
+    latest: "5.0.7"
+    latestReleaseDate: 2022-11-30
+    lts: false
+    discontinued: false
+  - releaseCycle: "4.13"
+    releaseDate: 2022-02-17
+    support: 2025-02-14
+    eol: 2026-02-14
+    latest: "4.13.14"
+    latestReleaseDate: 2022-11-30
+    lts: true
+    discontinued: false
+
+iconSlug: NA
+permalink: /contao
+releasePolicyLink: https://contao.org/en/release-plan.html
+releaseColumn: true
+releaseDateColumn: true
+versionCommand: composer show contao/core
```

Comment:
It seems there are multiple installation method available (https://docs.contao.org/manual/en/installation/install-contao/). Does the command `composer show contao/core` work whatever the installation method used ?

If no, I think it is best to remove the `versionCommand`.

Reply:
You are right. The Contao Manager is basically a GUI for composer and it is for users who cannot or do not want to use the CLI. So the command would still work, but it is obviously not reachable for these people.